Content.Server/Magic/MagicSystem.cs

```diff
@@ -115,7 +117,16 @@ private void AttemptLearn(EntityUid uid, SpellbookComponent component, UseInHand
     private void OnLearnComplete(EntityUid uid, SpellbookComponent component, LearnDoAfterComplete ev)
     {
         component.CancelToken = null;
-        _actionsSystem.AddActions(ev.User, component.Spells, uid);
+
+        if (!component.Persistent)
+            _actionsSystem.AddActions(ev.User, component.Spells, uid);
+        if (component.Persistent)
+            _actionsSystem.AddActions(ev.User, component.Spells, null);
+        if (component.SingleUse)
+        {
+            EntityManager.DeleteEntity(uid);
+            _popupSystem.PopupEntity(Loc.GetString("action-spellbook-crumble"), ev.User, Filter.Entities(ev.User), PopupType.Large);
```

Comment:
Should maybe be a caution popup?

Reply:
_popupSystem.PopupEntity(Loc.GetString("action-spellbook-crumble", ("book", uid)), ev.User, Filter.Entities(ev.User), PopupType.LargeCaution);